localtypings/pxteditor.d.ts

```diff
@@ -53,6 +53,7 @@ declare namespace pxt.editor {
         | "closeflyout"
         | "newproject"
         | "importproject"
+        | "importexternalproject"
```

Comment:
exportproject may be a better name?

Reply:
exportproject sound more like a message that would take an existing project in the makecode workspace and send a file back